src/contributor/dto/create-contributor.input.ts

```diff
@@ -0,0 +1,5 @@
+export class CreateContributorInput {
+    projectId: number;
+    userId: number;
+    role: string;
```

Comment:
Like that we can directly use the instance of this class as an input for our model.